### ISSUE TITLE
Documentation unclear

### ISSUE THREAD

**Issue by tkalmar** [nuxt/typescript#535]
**Describe the bug**
In https://typescript.nuxtjs.org/cookbook/plugins#plugin-2 an combined example is shown.
The used inject does not add `$myInjectedFunction` to the context.
I think the part:
```
 // nuxtContext.$myInjectedFunction
  interface Context {
    $myInjectedFunction(message: string): void
  }
```
is therefore wrong
either a direct setter of the `context.$myInjectedFunction` prop is missing, or the typing for context should be removed

**Comment by apgapg** [comment]
No its not wrong. Can you add the sample code of how you are using it?

**Comment by apgapg** [comment]
Yes the comment `// nuxtContext.$myInjectedFunction` must be removed or corrected in combined example

**Comment by rchl** [comment]
Nuxt's `inject` does actually inject to `Context` now. It has been like that for long enough now that it's OK to assume that it's always the case.
